src/contact.ts

```diff
@@ -61,6 +64,16 @@ export type ContactQueryFilter = {
 }
 
 /**
+ * @see https://github.com/Chatie/wechaty/blob/master/doc/webwxapp.js#L4057
```

Comment:
Please do not use `wechaty/doc`, because this directory is deprecated.

Should change refer to <https://github.com/Chatie/webwx-app-tracker> with the exactly commit hash.